libcudacxx/test/stdpar/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 3.21)
+
+set(CMAKE_CXX_COMPILER "/opt/nvidia/hpc_sdk/Linux_x86_64/25.1/compilers/bin/nvc++")
```

Comment:
This should be configured automatically by the devcontainer, it's not a good practice to hardcode compiler paths in a CMakeLists.txt.

We should probably just check that `CMAKE_CXX_COMPILER_ID STREQUAL NVHPC` and fail if not ([docs](https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_COMPILER_ID.html)).

Reply:
Oh sorry, that was from my local testing